packages/dev-server-api/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@react-native-community/dev-server-api",
+  "version": "4.7.0",
+  "license": "MIT",
+  "main": "build/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@react-native-community/cli-debugger-ui": "^4.2.1",
+    "@react-native-community/cli-tools": "^4.7.0",
+    "compression": "^1.7.1",
+    "connect": "^3.6.5",
+    "errorhandler": "^1.5.0",
+    "pretty-format": "^25.1.0",
+    "serve-static": "^1.13.1",
+    "ws": "^1.1.0"
```

Comment:
Would be awesome to migrate to the newest version as a followup

Reply:
Upgraded to the latest version of `ws` in another branch 👍